### ISSUE TITLE
Editted lines are not logged

### ISSUE THREAD

**Issue by irconan** [rawdigits/wee-slack#228]
Logs contain only original lines, no edits

**Comment by megalithic** [comment]
yeah, something changed recently, not sure if it's a slack/websocket thing or not, but i noticed this too. it seems like i'm seeing more wee-slack features failing to work as they were.

**Comment by trygveaa** [comment]
It's not possible to change a line after it has been written to the log, so we can't support this unless we write every change to the log.

@megalithic: This has never worked. Though, I have seen the edited line being stored instead of the original if the edit was done fast enough after the original message. However, this is not consistent, and I'm not sure in what case it happens.
